benches/all.rs

```diff
@@ -74,7 +74,7 @@ fn all(c: &mut Criterion) {
         // for performance comparisons.
         let mut group = c.benchmark_group("chars::from_byte_idx_std");
         for (text_name, text) in test_strings.iter() {
-            group.bench_function(format!("std::{}", text_name), |bench| {
+            group.bench_function(format!("std::{text_name}"), |bench| {
```

Comment:
This PR looks good to me except for these `format!()` changes, which IMO just feel like clippy suggesting to use fancier format features rather than something that actually improves the code.

I'd say revert these two `format!()` changes, and then the PR is good to go.

Thanks!

Reply:
Fixed